core/lib/multivm/src/versions/vm_latest/vm.rs

```diff
@@ -133,6 +133,7 @@ impl<S: WriteStorage, H: HistoryMode> VmInterface<S, H> for Vm<S, H> {
                 .map(GlueInto::glue_into)
                 .collect(),
             storage_refunds: self.state.storage.returned_refunds.inner().clone(),
+            pubdata_costs: Vec::new(),
```

Comment:
If I understand correctly you will change this line in the next PRs. Let's mark it with TODO to not forget to update, just in case